packages/dropdowns.next/src/elements/menu/ItemGroup.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import { useText } from '@zendeskgarden/react-theming';
+import { IItemGroupProps } from '../../types';
+import useMenuContext from '../../context/useMenuContext';
+import {
+  StyledItem,
+  StyledItemContent,
+  StyledItemGroup,
+  StyledItemTypeIcon,
+  StyledMenuSeparator
+} from '../../views';
+
+/**
+ * @extends LiHTMLAttributes<HTMLLIElement>
+ */
+export const ItemGroup = forwardRef<HTMLLIElement, IItemGroupProps>(
```

Comment:
@geotrev this whole `ItemGroup` concept is not a suggestion that we should have this API. I just wired this up in order to render the styled components for demo. We may just as well have `Menu`, `Item`, and `Separator` components as we do now 🤷. API TBD.

Reply:
As we discussed, in an ideal world this would ideally be a logic boundary, but not a big deal if that becomes reality and this needs to be rewritten.